e2e/test/scripts/create_cluster.sh

```diff
@@ -5,54 +5,48 @@ set -o pipefail
 set -o nounset
 set -x
 
-export LINODE_API_TOKEN="$1"
+export LINODE_TOKEN="$1"
 export CLUSTER_NAME="$2"
-export K8S_VERSION="$3"
+export KUBERNETES_VERSION="$3"
+export CAPL_VERSION="0.1.0"
+export WORKER_MACHINE_COUNT=1
+export LINODE_CONTROL_PLANE_MACHINE_TYPE=g6-standard-2
+export LINODE_MACHINE_TYPE=g6-standard-2
+export KUBECONFIG="$(realpath "$(dirname "$0")/../kind-management.conf")"
 
 if [[ -z "$4" ]]
 then
-  export REGION="eu-west"
+  export LINODE_REGION="eu-west"
 else
-  export REGION="$4"
+  export LINODE_REGION="$4"
 fi
 
-TEST_MANIFEST=$(realpath "$(dirname "$0")/../manifest/linode-blockstorage-csi-driver.yaml")
-
-cat > cluster.tf <<EOF
-variable "server_type_node" {
-  default = "g6-standard-2"
-}
-variable "nodes" {
-  default = 2
-}
-variable "server_type_master" {
-  default = "g6-standard-2"
-}
-variable "region" {
-  default = "$REGION"
-}
-variable "ssh_public_key" {
-  default = "${HOME}/.ssh/id_rsa.pub"
-}
-module "k8s" {
-  source  = "git::https://github.com/linode/terraform-linode-k8s.git?ref=master"
-  linode_token = "$LINODE_API_TOKEN"
-  cluster_name = "$CLUSTER_NAME"
-  csi_manifest = "file://${TEST_MANIFEST}"
-  server_type_node = "\${var.server_type_node}"
-  nodes = "\${var.nodes}"
-  server_type_master = "\${var.server_type_master}"
-  region = "\${var.region}"
-  ssh_public_key = "\${var.ssh_public_key}"
-  k8s_version = "$K8S_VERSION"
-}
-EOF
-
-terraform workspace new ${CLUSTER_NAME}
-
-terraform init
-
-terraform apply \
- -auto-approve
+kubectl create ns ${CLUSTER_NAME}
+(cd $(realpath "$(dirname "$0")"); clusterctl generate cluster ${CLUSTER_NAME} \
+  --target-namespace ${CLUSTER_NAME} \
+  --flavor clusterclass-kubeadm \
+  --config clusterctl.yaml \
+  | kubectl apply --wait -f -)
 
+c=8
+until kubectl get secret -n ${CLUSTER_NAME} ${CLUSTER_NAME}-kubeconfig; do
+  sleep $(((c--)))
+done
+
+kubectl get secret -n ${CLUSTER_NAME} ${CLUSTER_NAME}-kubeconfig -o jsonpath="{.data.value}" \
```

Comment:
`clusterctl get kubeconfig ${CLUSTER_NAME} > "$(pwd)/${CLUSTER_NAME}.conf"` should get the kubeconfig for you if you'd rather use clusterctl instead of the kubectl get secret